### PR TITLE
Add calculator features and styling

### PR DESCRIPTION
**Pull request by braincee**
_**In this pull request, the following requirements were implemented:**_

- I created a directory called components.
- Inside the components directory I created a new `Calculator` component
- I created a new `Calculator`.css file to add styling to the project.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (GzimAsani) [inline src/App.js]: - [ ] App.js should also be class-based. Kindly change this

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hi @braincee  
 
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job so far :clap: 
 
 ## Required Changes 
- [ ] _Check inline comments below :arrow_double_down: _


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by braincee** [comment]
Thanks very much, @elmar8287 and @GzimAsani 

thank-you-gif-6

**Pull request merged**